### PR TITLE
Rewrote the docs structure.

### PR DESCRIPTION
**Pull request by matvp91**
I've cleaned up the structure of the docs in order to better match a single page.

image

### CODE REVIEW

**Comment by sebmck** [comment]
I think we'll want to remove the manual cloning/build instructions. It's confused a lot of people. We should keep some of that only in the contributing docs. We actually already have a release published to npm under `rome` 🤐

**Comment by sebmck** [comment]
(That's just a general unrelated criticism of the current docs, your changes look great!)

File: website/src/docs/Installation.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
+Rome requires you to have `node` and `npm` installed on your system. Make sure you have these before you proceed.
 
-To install Rome, you must have `node` and `npm` installed on your system. If you do not have `node` and `npm`
-installed, install them before continuing.
-
-## Cloning and building
-
-Rome is not available via `npm` and must be installed from GitHub.
-In a folder of your choice, clone the `rome` repository:
+Rome is currently not available via `npm`, and must be installed from GitHub. In a folder of your choice, clone the `rome` repository.
```
Comment (KevinKelbie) [inline website/src/docs/Installation.md:5]: Rome technically is available through npm with `yarn add rome@^0.0.25` or `npm install rome@^0.0.25`. Not sure if its premature to add it in the docs though. Either way, I thought I'd point that out.

File: website/src/docs/Introduction.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Introduction
+
+Rome in an all-in-one JavaScript toolchain, including a compiler, linter, formatter, bundler, testing framework and more. It aims to be a comprehensive tool for anything related to the processing of JavaScript source code.
```
Comment (KevinKelbie) [inline website/src/docs/Introduction.md:3]:
```suggestion
Rome is an all-in-one JavaScript toolchain, including a compiler, linter, formatter, bundler, testing framework and more. It aims to be a comprehensive tool for anything related to the processing of JavaScript source code.
```

**Comment by matvp91** [comment]
Most of the documentation in this PR is straight out ported from the Docusaurus build, with the difference that the headings should better fit the single page structure.

Due to us moving to this format, the sidebar became a whole lot more important.

File: website/src/docs/Linting.md [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
+x description of what is not ok
+
+> 1 | the line of code that is problematic
+    |     ^^^ <the part of the line that is not ok>
+
+ ℹ helpful message(s) of what you can do to fix the problem
+ ```
 
-However if there is something not ok, `rome` will give you error messages in the following format:
+Let's look at example of what the linter might throw at you:
```
Comment (KevinKelbie) [inline website/src/docs/Linting.md:36]:
```suggestion
Let's look at an example of what the linter might throw at you:
```

**Pull request merged**